src/client/screens/ImportProjectScreen.tsx

```diff
@@ -33,39 +50,50 @@ const validate = (
   const regionConfig = importResource.data;
   const districtsDefinition = "resource" in importResource ? importResource.resource : null;
   const numberOfDistricts = form.numberOfDistricts;
+  const populationDeviation = form.populationDeviation || DEFAULT_POPULATION_DEVIATION;
   return numberOfDistricts &&
     maxDistrictId &&
     regionConfig &&
+    populationDeviation &&
     districtsDefinition &&
     numberOfDistricts >= maxDistrictId
-    ? { numberOfDistricts, regionConfig, districtsDefinition, valid: true }
-    : { numberOfDistricts, regionConfig, districtsDefinition, valid: false };
+    ? { numberOfDistricts, regionConfig, districtsDefinition, populationDeviation, valid: true }
+    : { numberOfDistricts, regionConfig, districtsDefinition, populationDeviation, valid: false };
 };
 
 type ImportResource = WriteResource<IRegionConfig | null, DistrictsDefinition>;
 
 interface ConfigurableForm {
   readonly numberOfDistricts: number | null;
+  readonly isCustom?: boolean;
```

Comment:
I don't think `isCustom` should ever be `null` / `undefined` right? - the checkbox is either checked or not.
```suggestion
  readonly isCustom: boolean;
```